docs/user/resources/01-serverless.md

```diff
@@ -0,0 +1,104 @@
+| **conditions.&#x200b;reason** (required) | string | Contains a programmatic identifier indicating the reason for the condition's last transition. Producers of specific condition types may define expected values and meanings for this field and whether the values are considered a guaranteed API. The value should be a camelCase string. This field may not be empty. |
+| **conditions.&#x200b;status** (required) | string | status of the condition, one of `True`, `False`, `Unknown`. |
+| **conditions.&#x200b;type** (required) | string | type of condition in camelCase or in `foo.example.com/CamelCase`. --- Many **.condition.type** values are consistent across resources like `Available`, but because arbitrary conditions can be useful (see **.node.status.conditions**), the ability to deconflict is important. The regex it matches is `(dns1123SubdomainFmt/)?(qualifiedNameFmt)` |
+| **dockerRegistry**  | string | Used registry configuration. Contains registry URL or "internal" |
+| **eventingEndpoint**  | string | Used Eventing endpoint. |
+| **served** (required) | string | Served signifies that current Serverless is managed. Value can be one of `True`, or `False`. |
+| **state**  | string | Signifies the current state of Serverless. Value can be one of `Ready`, `Processing`, `Error`, or `Deleting`. |
+| **tracingEndpoint**  | string | Used Tracing endpoint. |
+
+<!-- TABLE-END -->
+
+### Status reasons
+
+Processing of a Serverless CR can succeed, continue, or fail for one of these reasons:
+
+
+# Serverless CR conditions
+
+This section describes the possible states of the Serverless CR. Three condition types, `Installed`, `Configured` and `Deleted`, are used.
+
+| No | CR State   | Condition type | Condition status | Condition reason      | Remark                                        |
+|----|------------|----------------|------------------|-----------------------|-----------------------------------------------|
+| 1  | Ready      | Configured     | true             | Configured            | Serverless configuration verified             |
+| 2  | Processing | Configured     | unknown          | ConfigurationCheck    | Serverless configuration verification ongoing |
+| 3  | Error      | Configured     | false            | ConfigurationCheckErr | Serverless configuration verification error   |
+| 4  | Ready      | Installed      | true             | Installed             | Serverless workloads deployed                 |
+| 5  | Processing | Installed      | unknown          | Installation          | Deploying serverless workloads                |
+| 6  | Error      | Installed      | false            | InstallationErr       | Deployment error                              |
+| 7  | Error      | Installed      | false            | ServerlessDuplicated  | Only one Serverless CR is allowed             |
```

Comment:
The ServerlessDuplicated reason occurs only with the `Configured` type. I would suggest to re-order this element and put it together with other Configured typed conditions

```suggestion
| 7  | Error      | Configured     | false            | ServerlessDuplicated  | Only one Serverless CR is allowed             |
```

Reply:
done